src/components/dashboard/Dashboard.js

```diff
@@ -1,10 +1,62 @@
 import React from 'react'
 import './Dashboard.css';
+import FullCalendar from '@fullcalendar/react'
+import listPlugin from '@fullcalendar/list'
+import {ListWrapper} from './ListWrapper'
+
 
 const Dashboard = () => {
   return (
     <div className="Dashboard">
       <h1>THIS IS THE Dashboard</h1>
+      <ListWrapper>
+        <FullCalendar
```

Comment:
Awesomeness! Glad you were able to figure the upcoming events view. Looks great, and it's perfect for UX on dashboard!